src/views/questionView.js

```diff
@@ -12,6 +13,7 @@ export const createQuestionElement = (question) => {
 
   // I use String.raw just to get fancy colors for the HTML in VS Code.
   element.innerHTML = String.raw`
+    <p id="${COUNTDOWN_TIME_ID}"></p>
```

Comment:
Ideally, we should create a new view for counter.
Could you please check if it's easy to do, otherwise let's keep it like this an we can refactor this part later on.